### ISSUE TITLE
[new_frontend] Main navigation menu disappears at 767px width (or narrower)

### ISSUE THREAD

**Issue by pdehaan** [mozilla-services/FindMyDevice#35]
Found in d919f64a6ad4230c6eb5561acb65d45adea3c24e

---

**tl;dr** Breakpoints gone wild?
### Steps to reproduce:
1. `$ grunt serve`
2. Resize your browser to <=767px wide.
### Actual results:

The main menu disappears and presumably jumps to the bottom (a black bar appears at least). But menu options don't reappear, meaning I cannot erase or play sounds or third option.

![find_my_device](https://cloud.githubusercontent.com/assets/557895/3003733/cabea88e-dd5e-11e3-9c04-93f36b9d8993.png)
**Figure 1.** Page at 767px wide.

Actually, it looks like the bottom black bar may be a 33% 33% 33% menubar, just without icons or text, so it just lacks some visual styling.

For reference, here is the same page, but 1px wider (768px):
![find_my_device](https://cloud.githubusercontent.com/assets/557895/3003730/b38ca1de-dd5e-11e3-8bd7-da4cecc0e7b0.png)
**Figure 2.** Page at 768px wide.
### Expected results:

Icon colors and text at smaller breakpoint.
